roles/istio/tasks/certificates.yaml

```diff
@@ -0,0 +1,137 @@
+---
+
+################### Start Get Istio External IP ###################
+
+- name: Wait for External IP Address to be ready
+  community.kubernetes.k8s_info:
+    api_version: v1
+    kind: Service
+    name: istio-ingressgateway
+    namespace: istio-system
+    wait: yes
+    wait_timeout: 360
+  register: istio_web_service
+  until: >
+    'ingress' in istio_web_service.resources[0].status.loadBalancer and
+    istio_web_service.resources[0].status.loadBalancer.ingress[0].ip != ""
+  retries: 20
+  delay: 5
+
+
+- name: Set fact about Istio Ingress Gateway External IP
+  set_fact:
+    istio_external_ip: "{{ istio_web_service.resources[0].status.loadBalancer.ingress[0].ip }}"
```

Comment:
We're using this operation quite often. Let's open issue to explore how to add it as part of the collection's built-in so we can benefit from code re-usability